extension/js/service_worker/background.ts

```diff
@@ -67,4 +67,7 @@ console.info('background.js service worker starting');
   BrowserMsg.alarmListen();
   await BgHandlers.updateUninstallUrl({});
   injectFcIntoWebmail();
+
+  // Thunderbird event handlers
+  BrowserMsg.thunderbirdSecureComposeHandler();
```

Comment:
it should be performed only when extension is opened in Thunderbird, as currently in Chrome it causes errors:
```
catch.js:64 ReferenceError: browser is not defined
    at BrowserMsg.thunderbirdSecureComposeHandler (chrome-extension://pjohdcpiapjhagmakkjpgjmodmgfmkkd/js/common/browser/browser-msg.js:284:9)
    at chrome-extension://pjohdcpiapjhagmakkjpgjmodmgfmkkd/js/service_worker/background.js:82:20
    at <async> <anonymous> (/extension/js/service_worker/background.ts:20:2)
```

Reply:
checks added - thank you!